utils/flags.go

```diff
@@ -39,7 +39,8 @@ type WskDeployFlags struct {
 	Strict           bool // strict flag to support user defined runtime version.
 	Key              string
 	Cert             string
-	Managed          bool // OpenWhisk Managed Deployments
+	Managed          bool   // OpenWhisk Managed Deployments
```

Comment:
@pritidesai do we need to use/key off this flag elsewhere?

Reply:
yup @mrutkows we have used it in many places, @lionelvillard hasn't introduced a new flag, its just a go fmt playing with spaces and moving flags around :)